src/main/webapp/search-interview.js

```diff
@@ -54,48 +66,76 @@ function selectInterview(interviewer) {
   }
   let utcStartTime = interviewer.getAttribute('data-utc');
   let position = document.getElementById('position').value;
-  if (confirm(
-      `You selected: ${date} from ${time} with a ` +
-      `${company} ${job}. ` +
-      `Click OK if you wish to proceed.`)) {
-    alert(
-      `You have scheduled a ${position} interview on ${date}` +
-      ` from ${time} with a ${company} ` +
-      `${job}. Check your email for more ` +
-      `information.`);
-    if (company === '<Not specified>') {
-      company = '';
+  let role = selectedRole();
+  if (role === 'Interviewee') {
+    if (confirm(
+        `You selected: ${date} from ${time} with a ` +
+        `${company} ${job}. ` +
+        `Click OK if you wish to proceed.`)) {
+      alert(
+        `You have scheduled a ${position} interview on ${date}` +
+        ` from ${time} with a ${company} ` +
+        `${job}. Check your email for more ` +
+        `information.`);
+      updateDatastore('POST', company, job, utcStartTime);
     }
-    if (job === '<Not specified>') {
-      job = '';
+  }
+  if (role === 'Shadow') {
```

Comment:
probably should be `else if`, not a big deal though.

Reply:
Changed